Library/Formula/c.rb

```diff
@@ -0,0 +1,17 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.07.tar.gz"
+  sha1 "c40044a2747b97a2f8b550c1d417bb03848fcd40"
+
+  depends_on "gnu-sed"
```

Comment:
It looked like the upstream commit made this not need `gnu-sed` any more? Is it possible for this to be made to work with BSD `sed` instead? Thanks!